config/settings.js

```diff
@@ -38,6 +38,7 @@ const SCHEMA_PIPELINE_SETTINGS = Joi.object()
         showEventTriggers: Joi.boolean().optional(),
         filterEventsForNoBuilds: Joi.boolean().optional(),
         aliasName: Joi.string()
+            .allow('')
```

Comment:
does it need to be empty or optional settings?

Reply:
I see `.optional()` on line 45, so should be okay I think
just wondering if you mean `.allow(null)` vs `.allow('')`

Reply:
We want user to be able change back aliasname to empty string, so I believe `.allow('')` will be good, what do you think on this?

Reply:
sure